src/components/mobile/mobile.vue

```diff
@@ -33,8 +33,18 @@
       </div>
     </template>
 
-    <template #toolbar v-if="$x.query.search">
-      <MobileToolbar class="x-padding--left-05 x-padding--bottom-05 x-padding--right-03" />
+    <template #toolbar v-if="$x.query.search && !$x.redirections.length">
+      <div>
```

Comment:
The `div` is not taking the full width

```suggestion
      <div class="x-list__item--expand">
```

![image](https://user-images.githubusercontent.com/77450928/203002457-ad84ad38-7c39-495f-af71-4f2ba5eb6fc9.png)